Content.Server/_Wega/Mining/MiningServerSystem.cs

```diff
@@ -32,7 +32,20 @@ public override void Update(float frameTime)
         while (query.MoveNext(out var uid, out var server, out var consumer))
         {
             if (server.IsBroken || consumer.ReceivedPower < server.ActualPowerConsumption)
-            {
+            {   
+                if (server.IsBroken)
+                {
+                    if (consumer.DrawRate != 0f)
+                        consumer.DrawRate = 0f;
+
+                    if (server.IsActive)
+                    {
+                        server.IsActive = false;
+                        UpdateAppearance(uid, server);
+                        _ambient.SetAmbience(uid, false);
+                    }
+                }
```

Comment:
Логичнее тогда уж было сделать так, чтобы отключало и при перебое питания и при сломанном состоянии, чем пихать по 10 проверок одного и того же(я преувеличиваю если что, просто подмечаю)

Ну и зачем проверка на 0...
```suggestion
                consumer.DrawRate = 0f;

                if (server.IsActive)
                {
                    server.IsActive = false;
                    UpdateAppearance(uid, server);
                    _ambient.SetAmbience(uid, false);
                }
```